src/Turbo/src/TurboBundle.php

```diff
@@ -45,6 +46,24 @@ public function process(ContainerBuilder $container): void
                 }
             }
         }, PassConfig::TYPE_BEFORE_REMOVING);
+
+        $container->addCompilerPass(new class() implements CompilerPassInterface {
+            public function process(ContainerBuilder $container): void
+            {
+                $serviceIds = [
+                    ...$container->findTaggedServiceIds('turbo.broadcaster'),
+                    ...$container->findTaggedServiceIds('turbo.renderer.stream_listen'),
+                ];
+
+                foreach ($serviceIds as $serviceId => $tags) {
+                    $definition = $container->getDefinition($serviceId);
+
+                    $definition
+                        ->addArgument(new Reference('turbo.id_formatter'))
+                        ->addArgument(new Reference('turbo.doctrine_class_resolver'));
+                }
+            }
+        });
```

Comment:
Those tags refer to BroadcasterInterface and TurboStreamListenRendererInterface services, so you cannot add those arguments there as it would potentially impact other services which don't use them.